addons/snailmail_account/wizard/account_invoice_send.py

```diff
@@ -82,14 +86,14 @@ def notify_invalid_addresses(self):
         self.env['bus.bus'].sendone(
             (self._cr.dbname, 'res.partner', self.env.user.partner_id.id),
             {'type': 'snailmail_invalid_address', 'title': _("Invalid Addresses"),
-            'message': _("%s of the selected invoice(s) had an invalid address and were not sent", self.invalid_addresses)}
+            'message': _("%s of the selected invoice(s) had an invalid address and were not sent", self.invalid_invoices)}
         )
 
     def invalid_addresses_action(self):
         return {
             'name': _('Invalid Addresses'),
             'type': 'ir.actions.act_window',
             'view_mode': 'kanban,tree,form',
-            'res_model': 'account.move',
-            'domain': [('id', 'in', self.mapped('invalid_invoice_ids').ids)],
+            'res_model': 'res.partner',
+            'domain': [('id', 'in', self.mapped('invalid_partner_ids').ids)],
```

Comment:
why not just self.invalid_partner_ids.ids ? The mapped was a mistake in the old code.